### PR TITLE
Several changes for --wotta option

### PR DESCRIPTION
**Pull request by cwotta**
Revamped MCMC_output() class
Updated saved info to include --wotta option info

### CODE REVIEW

File: pyigm/metallicity/mcmc.py [hunk 1]
```diff
@@ -459,17 +490,26 @@ def plotinfo(self, sampler, use_pkl=False):
...
-        #now do a corner plot
+
+        ######
+        ##Corner Plot
+        ######
+        
         samples = sampler.chain[:,self.burn:, :].reshape((-1,self.ndim))
-        cfig = corner.corner(samples, labels=self.mod_axistag, quantiles=[0.05,0.5,0.95],verbose=False)
-        cfig.text(0.33,0.90,self.info['name']+", UVB="+self.UVB+", logUconstraint="+str(self.logUconstraint),horizontalalignment='left',fontsize=10)
+        cfig = corner.corner(samples, labels=self.mod_axistag, label_kwargs = {"fontsize": 16}, quantiles=[0.05,0.5,0.95],verbose=False)
```
Comment (profxj) [inline pyigm/metallicity/mcmc.py:504]: Is this for an updated corner method?

File: pyigm/scripts/pyigm_mtlmcmc.py [hunk 2]
```diff
@@ -142,14 +174,16 @@ def run_mcmc_wotta(args):
     ##Test whether to include the carbalpha parameter, and use the correct UVB
     ##...But set a default first
     if not args.grid:
-        args.grid = 'grid_minimal'
+        args.grid = '/afs/crc.nd.edu/group/CGMND/Cloudy_grids/grid_cgm_extensive'
```
Comment (profxj) [inline pyigm/scripts/pyigm_mtlmcmc.py]: Let's not hard code this.  Can you have it be 
read in as a parameter?

**Review by profxj — COMMENTED** [review]
Only 2 minor comments.
Address these and I'll merge.

**Comment by profxj** [comment]
And thanks for this.  
I love all the documentation!

File: pyigm/metallicity/mcmc.py (hunk 1 above)
Comment (cwotta) [reply pyigm/metallicity/mcmc.py:504]: No, this is the same plot.  I'm just adding (1) a plot title (to match the other plots) and (2) adjusting font size of the axis titles (x and y labels).  I now have the plot title as a variable, too, so it's defined in a single place.

File: pyigm/scripts/pyigm_mtlmcmc.py (hunk 2 above)
Comment (cwotta) [reply pyigm/scripts/pyigm_mtlmcmc.py]: It *is* read in as a parameter (see, e.g., docs/LLS_Metallicities-Wotta/queue_script.sh).  This line just sets a default if it doesn't receive an argument.

Comment (cwotta) [reply pyigm/scripts/pyigm_mtlmcmc.py]: (see also the documentation at the bottom of the script for input arguments)

**Comment by profxj** [comment]
Merging.

**Pull request merged**